### DISCUSSION TITLE
Customize date format

### DISCUSSION THREAD

**Discussion by b3nj5m1n** [athensresearch/athens#1329] [Feature Requests]
I'd like to be able to customize the date format, so for example instead of the current MDY "March 20, 2021" have it displayed as "20.3.2021" or "20th March 2021".

It's a lot more familiar to me, and I'm sure for many others, as the MDY format is relatively uncommon. ([See Date format by country](https://en.wikipedia.org/wiki/Date_format_by_country))

Just to be clear, this is a pretty small thing and doesn't have much of an impact on the user experience, it would just be a nice feature to have eventually.

**Comment by latvia234** [comment]
I would like an option to change it to:
yyyy-MM-dd
2021-08-15
